libs/message-handler/src/protocols/forward-client.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  DIDDoc,
+  DIDResolver,
+  IMessage,
+  Message,
+  SecretsResolver,
+} from 'didcomm';
+import { v4 as uuidv4 } from 'uuid';
+import axios from 'axios';
+import PeerDIDResolver from '../../../did-resolver-lib/src/resolver';
+import {
+  ExampleDIDResolver,
+  ExampleSecretsResolver,
+} from '../../../did-resolver-lib/src/ExampleDIDResolver';
+import { FROM } from '../did_doc/client';
+import { ROUTING } from '../shared_data/message_types';
+import { CLIENT_SECRETS } from '../secrets/client';
+import { MEDIATOR_ENDPOINT } from '../shared_data/endpoints';
+import { CONTENT_TYPE } from '../shared_data/constant';
+
+export function buildMessage(to: string[], message: unknown): Message {
```

Comment:
why do you use the unknown type here?

Reply:
> why do you use the unknown type here?

Its because the message could be of any type, Array, string, etc but using any will make eslint complain

Reply:
> Its because the message could be of any type, Array, string, etc but using any will make eslint complain

You need a way to handle various input types within the function. If the function can accept any type of input, it should include logic to process these variations accordingly. It's not clear how your function addresses this.

Reply:
because the message can be of any type

Reply:
> because the message can be of any type

Read my previous comment again please.